recipes/domdf-python-tools/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "domdf-python-tools" %}
+{% set version = "3.2.2.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/domdf-python-tools-{{ version }}.tar.gz
+  sha256: fef90a0109b284424b14f9513e3b543f68feb1d48beddde435928e0f05f4664e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - importlib-metadata >=3.6.0  # [py<39]
+    - importlib-resources >=3.0.0  # [py<37]
```

Comment:
I believe these are installable on any python and we can drop the selectors to make this noarch.

Reply:
Yes, although they're only *required* on older Python versions as the imports as guarded with a version check. If it's better to make the package noarch I'll drop the selectors.